.github/renovate.json

```diff
@@ -1,10 +1,10 @@
 {
 	"extends": [ "config:base" ],
-	"labels": [ "[Type] Janitorial", "[Status] Needs Review" ],
+	"labels": [ "[Type] Janitorial" ],
```

Comment:
If they're not tagged for review, will we remember to look at them?

Reply:
That's a good question. On one hand, I would not want to "pollute" the list of PRs pending review by adding PRs that we know we won't look at for a whole month.
On the other hand, I wouldn't want those PRs to go ignored either.

I'm not sure what's the best approach there.

@kraftbj What would be your take on this?

Reply:
As long as have some system to make sure they are seen, I think not having them automatically marked for review is fine. 

That system could be as simple as the release lead clicking a link to display all renovate Open PRs not labeled "in progress", selecting all and marking for review right after the branch cut. Could also automate this with GH CLI etc.

Reply:
For now I've added it back, we can revisit later.